kinto/config/__init__.py

```diff
@@ -0,0 +1,54 @@
+import os
+import binascii
+import codecs
+from six.moves import input
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+def render_template(template, destination, **kwargs):
+    template = os.path.join(HERE, template)
+
+    with codecs.open(template, 'r', encoding='utf-8') as f:
+        raw_template = f.read()
+        rendered = raw_template.format(**kwargs)
+        with codecs.open(destination, 'w+', encoding='utf-8') as output:
+            output.write(rendered)
+
+
+def init(config_file):
+    values = {}
+    values['secret'] = binascii.b2a_hex(os.urandom(32))
+
+    backend = input("Which backend to use? "
+                    "(1 - postgresql, 2 - redis, default - memory) ").strip()
```

Comment:
default when empty string is seized yes, but surrounding the input with a `while True: ... if backend in ('', '1', '2'): break` would not change much